binary_search_tree/binary_search_tree.py

```diff
@@ -5,10 +5,42 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+    element=self
+    while True:
+      if value<element.value:
+        if element.left==None:
+          element.left=BinarySearchTree(value)
+          break
+        else:
+          element=element.left
+      else: 
+        if element.right==None:
+          element.right=BinarySearchTree(value)
+          break
+        else:
+          element=element.right
+
 
   def contains(self, target):
-    pass
+    element=self
+    while True:
+      if element.value==target:
+        return True
+      elif target<element.value:
+        if element.left is not None:
```

Comment:
You've been using `==` when checking for `None` up to this point.  PEP8 prefers `is` 

> Comparisons to singletons like None should always be done with is or is not, never the equality operators

https://www.python.org/dev/peps/pep-0008/#programming-recommendations